l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -75,7 +81,9 @@ contract L1NativeTokenVault is IL1NativeTokenVault, Ownable2StepUpgradeable, Pau
             uint256 balanceBefore = address(this).balance;
             L1_SHARED_BRIDGE.transferTokenToNTV(_token);
             uint256 balanceAfter = address(this).balance;
-            require(balanceAfter > balanceBefore, "NTV: 0 eth transferred");
+            if (balanceAfter == balanceBefore) {
```

Comment:
maybe <= and not ==

Reply:
this is for eth token, so doesnt matter

Reply:
and `==` is more adherent with the error message, but let's do the `<=` one just in case